scheduler/src/test/java/org/cloudfoundry/autoscaler/scheduler/dao/ScheduleDaoImplTest.java

```diff
@@ -1,47 +1,110 @@
 package org.cloudfoundry.autoscaler.scheduler.dao;
 
+import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertNotNull;
 
+import java.util.List;
+
 import javax.transaction.Transactional;
 
-import org.apache.commons.logging.Log;
-import org.apache.commons.logging.LogFactory;
 import org.cloudfoundry.autoscaler.scheduler.entity.ScheduleEntity;
-import org.cloudfoundry.autoscaler.scheduler.util.DataSetupHelper;
+import org.cloudfoundry.autoscaler.scheduler.util.TestDataSetupHelper;
+import org.junit.After;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.test.annotation.Rollback;
-import org.springframework.test.context.ContextConfiguration;
+import org.springframework.boot.test.context.SpringBootTest;
 import org.springframework.test.context.junit4.SpringRunner;
 
 /**
- * @author Fujitsu
+ * 
  *
  */
 @RunWith(SpringRunner.class)
-@ContextConfiguration(locations = { "classpath:applicationContext-test.xml" })
-@Rollback(true)
-
+@SpringBootTest
 public class ScheduleDaoImplTest {
 
 	@Autowired
-	ScheduleDao scheduleDao;
-	private Log logger = LogFactory.getLog(this.getClass());
+	private ScheduleDao scheduleDao;
+
+	private String appId = TestDataSetupHelper.getAppId_1();
 
+	@After
+	@Transactional
+	public void afterTest() {
+		for (ScheduleEntity entity : scheduleDao.findAllSchedulesByAppId(appId)) {
+			scheduleDao.delete(entity);
+		}
+	}
+	
 	@Test
 	@Transactional
-	public void testCreateSchedule_01() {
-		logger.info("Executing Test Create Schedule to create one schedule ...");
-		ScheduleEntity entity = DataSetupHelper.generateScheduleEntity();
+	public void testFindAllSchedulesByAppId_01() {
```

Comment:
Do not number them. Name them based on what it is testing.
findAllSchedulesByAppId_with_no_schedules()